src/grpc_generated/go/README.rst

```diff
@@ -27,31 +27,32 @@
 
 |License|
 
+Regenerate Go Client stubs
+==========================
+  # Clone the repo containing the proto definitions, from within client/src/grpc_generated/go/
+    git clone https://github.com/triton-inference-server/common.git
+
+  # Compiles *.proto to *.pb.go
+  ./gen_go_stubs.sh
+
+
 Example Go Client
 =================
 
-This sample script assumes that the protobuf stubs will be compiled
-local to the client, under the nvidia_inferenceserver directory. This
-can be easily tweaked by uncommenting gen_go_stubs.sh to generate the
-stubs in ${GOPATH}/src/nvidia_inferenceserver to have something more
-global instead.
-
-Usage::
+To regenerate the grpc client::
 
-  # Clone repos
+  # Clone the repo containing the example model, from within client/src/grpc_generated/go/.
   git clone https://github.com/triton-inference-server/server.git
-  git clone https://github.com/triton-inference-server/common.git
 
   # Setup "simple" model from example model_repository
   cd server/docs/examples
   ./fetch_models.sh
 
   # Launch (detached) Triton
   docker run -d -p8000:8000 -p8001:8001 -p8002:8002 -it -v $(pwd)/model_repository:/models nvcr.io/nvidia/tritonserver:20.11-py3 tritonserver --model-store=/models
```

Comment:
Hi @NikeNano, thanks for the contribution!

Can you verify this works in the latest Triton container such as 22.07, if so can you update this line too?

Reply:
Happy to help out, great work with Triton really like the experience so far :)